### ISSUE TITLE
npm install has failed. Tarball link is broken

### ISSUE THREAD

**Issue by smelukov** [npm/npm#13669]
#### I'm opening this issue because:
- [ ] npm is crashing.
- [x] npm is producing an incorrect install.
- [ ] npm is doing something I don't understand.
- [ ] Other (_see below for feature requests_):
#### What's going wrong?

`npm ERR! fetch failed http://registry.npmjs.org/basisjs-tools-build/-/basisjs-tools-build-1.8.0.tgz`
#### How can the CLI team reproduce the problem?

`npm i basisjs-tools`
### supporting information:
- `npm -v` prints: `3.10.6`
- `node -v` prints: `6.3.1`
- `npm config get registry` prints: `http://registry.npmjs.org/`
- Windows, OS X, or Linux?: `OS X`
- Network issues:
  - Geographic location where npm was run:
  - [ ] I use a proxy to connect to the npm registry.
  - [ ] I use a proxy to connect to the web.
  - [ ] I use a proxy when downloading Git repos.
  - [ ] I access the npm registry via a VPN
  - [ ] I don't use a proxy, but have limited or unreliable internet access.
- Container:
  - [ ] I develop using Vagrant on Windows.
  - [ ] I develop using Vagrant on OS X or Linux.
  - [ ] I develop / deploy using Docker.
  - [ ] I deploy to a PaaS (Triton, Heroku).

**Comment by smelukov** [comment]
I've trying to `npm i basisjs-tools` from different devices and places and got the same result - error

**Comment by smelukov** [comment]
`npm info basisjs-tools-build`:
```json
{
   name:'basisjs-tools-build',
   description:'Build tools for basis.js framework',
   'dist-tags':{
      latest:'1.8.0'
   },
   versions:[
      '1.0.0',
      '1.0.1',
      '1.0.2',
      '1.1.0',
      '1.2.0',
      '1.2.1',
      '1.3.0',
      '1.3.1',
      '1.3.2',
      '1.4.0',
      '1.4.1',
      '1.5.0',
      '1.5.1',
      '1.6.0',
      '1.7.0',
      '1.7.1',
      '1.7.2',
      '1.8.0'
   ],
   maintainers:[
      'lahmatiy <rdvornov@gmail.com>'
   ],
   time:{
      modified:'2016-08-10T15:01:53.707Z',
      created:'2015-11-08T20:02:08.095Z',
      '1.0.0':'2015-11-08T20:02:08.095Z',
      '1.0.1':'2015-11-10T12:25:15.160Z',
      '1.0.2':'2015-11-22T17:38:22.046Z',
      '1.1.0':'2015-11-23T15:46:56.783Z',
      '1.2.0':'2016-01-19T13:49:40.562Z',
      '1.2.1':'2016-01-21T10:31:22.761Z',
      '1.3.0':'2016-02-16T15:28:41.114Z',
      '1.3.1':'2016-02-16T18:59:17.483Z',
      '1.3.2':'2016-02-16T19:24:34.607Z',
      '1.4.0':'2016-03-19T23:23:53.611Z',
      '1.4.1':'2016-03-20T00:11:57.115Z',
      '1.5.0':'2016-03-24T23:04:40.570Z',
      '1.5.1':'2016-03-24T23:26:49.285Z',
      '1.6.0':'2016-06-24T12:18:05.988Z',
      '1.7.0':'2016-07-15T15:45:05.191Z',
      '1.7.1':'2016-07-18T10:52:11.893Z',
      '1.7.2':'2016-07-21T10:19:59.848Z',
      '1.8.0':'2016-08-10T15:01:53.707Z'
   },
   homepage:'https://github.com/basisjs/basisjs-tools',
   repository:{
      type:'git',
      url:'git+https://github.com/basisjs/basisjs-tools-build.git'
   },
   contributors:[
      'Roman Dvornov <rdvornov@gmail.com>'
   ],
   author:'Roman Dvornov <rdvornov@gmail.com>',
   bugs:{
      url:'https://github.com/basisjs/basisjs-tools-build/issues'
   },
   license:'MIT',
   readmeFilename:'README.md',
   title:'Basis.js build tools',
   version:'1.8.0',
   main:'./lib/index.js',
   engines:{
      node:'>=0.12.0'
   },
   eslintConfig:{
      env:{
         node:true,
         mocha:true,
         es6:true
      },
      rules:{
         'no-undef':2,
         'no-unused-vars':[
            2,
            {
               vars:'all',
               args:'after-used'
            }
         ]
      }
   },
   scripts:{
      test:'mocha',
      'test-code':'jscs lib && eslint lib',
      'test-all':'npm run test-code && npm test',
      travis:'npm run test-all'
   },
   dependencies:{
      'basisjs-tools-ast':'~1.4.0',
      'basisjs-tools-config':'~1.1.0',
      chalk:'~1.1.1',
      clap:'~1.1.0',
      'es6-promise-polyfill':'^1.2.0',
      'fixed-width-string':'^1.0.0',
      mime:'~1.3.4',
      minimatch:'^3.0.2',
      resolve:'^1.1.7',
      seedrandom:'~2.4.2',
      'uglify-js':'1.3.5'
   },
   devDependencies:{
      eslint:'^2.2.0',
      jscs:'~2.11.0',
      mocha:'~2.3.4'
   },
   files:[
      'bin',
      'lib',
      'LICENSE',
      'README.md'
   ],
   gitHead:'50cbb4c3a8cdcb4c0081882d658e71637dec4dbb',
   dist:{
      shasum:'af0237e3437b9af34e1482088d49db3a4af7a129',
      tarball:'https://registry.npmjs.org/basisjs-tools-build/-/basisjs-tools-build-1.8.0.tgz'
   },
   directories:{

   }
}
```
but tarball link https://registry.npmjs.org/basisjs-tools-build/-/basisjs-tools-build-1.8.0.tgz is broken
